### DISCUSSION TITLE
Dizideki sayılardan en büyük sayıyı bulma sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#19] [Soru Cevap]
Verilen sonlu bir (123) dizisinin her seferinde başından ya da sonundan bir eleman alınarak yeni bir sayı dizisi oluşturuluyor. Yeni dizi oluşturulurken orijinal diziden alınan sayı başa ya da sona konuyor ve bu sayı orijinal diziden siliniyor. Örneğin, ilk dizi orijinal diziyi göstermek üzere (122312, ) , (12231, 2)  , (2231,21) , (223, 121) şeklinde devam ediliyor. 1231132312 dizisinden bu şekilde elde edilecek en büyük sayının son üç basamağı nedir?

**Comment by akgembi12** [comment]
en büyük sayı 3332221111 olur. son 3 basamak da 111 olur

**Reply by sonsuzus** [reply]
Verilen diziden bu dizi elde edilebiliyor mu?
